runtime/init.go

```diff
@@ -298,12 +298,19 @@ func (d *Default) decodeJoinCmd(cmd string) {
 	stringSlice := strings.Split(cmd, " ")
 
 	for i, r := range stringSlice {
-		switch r {
-		case "--token":
+		// upstream error, delete \t, \\, \n, space.
+		r = strings.ReplaceAll(r, "\t", "")
+		r = strings.ReplaceAll(r, "\n", "")
+		r = strings.ReplaceAll(r, "\\", "")
+		r = strings.TrimSpace(r)
+		logger.Debug("[####]%d :%s:", i, r)
```

Comment:
delete this logger or show the joinToken hash ...

Reply:
show the joinToken hash.